src/model/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'user' })
```

Comment:
Better to have the `version` indicated in the table name because it is possible that you will have a second, third, ... more versions of User during your implementation, and the version number can easily help differentiate them.

In short, I will rename to `user_v1`